File: executable/r/data-visible.r
library(plotly)

p <- plotly(username='test-runner', key='9h29fe3l0x')

trace1 <- list(
  x = c(0, 1, 2, 3), 
  y = c(0, 2, 4, 6), 
  visible = TRUE, 
  type = "scatter"
)
trace2 <- list(
  x = c(0, 1, 2, 3), 
  y = c(8, 4, 2, 0), 
  visible = FALSE, 
  type = "scatter"
)
data <- list(trace1, trace2)

response <- p$plotly(data, kwargs=list(filename="data-visible", fileopt="overwrite"))
url <- response$url
filename <- response$filename

File: executable/node/data-visible.js
var plotly = require('plotly')('test-runner', '9h29fe3l0x')

var trace1 = {
  x: [0, 1, 2, 3], 
  y: [0, 2, 4, 6], 
  visible: true, 
  type: "scatter"
};
var trace2 = {
  x: [0, 1, 2, 3], 
  y: [8, 4, 2, 0], 
  visible: false, 
  type: "scatter"
};
var data = [trace1, trace2];

var graph_options = {filename: "data-visible", fileopt: "overwrite"}
plotly.plot(data, graph_options, function (err, msg) {
    console.log(msg);
});